web/themes/new_weather_theme/assets/js/charts/hourly-pops.js

```diff
@@ -1,98 +1,86 @@
 /* global Chart ChartDataLabels */
-(() => {
```

Comment:
Looks like we have some charting stuff that was accidentally included here

Reply:
Not an accident! These were merged into main a little bit ago, so I needed to tweak them for the bundling to work correctly with them.